dbms/src/Flash/Coprocessor/ExecutionSummaryCollector.cpp

```diff
@@ -99,6 +106,12 @@ void ExecutionSummaryCollector::addExecuteSummaries(tipb::SelectResponse & respo
             }
             current.concurrency++;
         }
+        // get execution info from dag context's scan_context
+        if (const auto & iter = dag_context.scan_context_map.find(executor_id); iter != dag_context.scan_context_map.end())
```

Comment:
Maybe `auto fill_execution_summary = [&](const String & executor_id, const BlockInputStreams & streams, const ScanContextMap & scan_context_map)` is better.